apps/console/src/public/deployment.config.json

```diff
@@ -331,7 +331,9 @@
                         "internal_role_mgt_view"
                     ],
                     "delete": []
-                }
+                },
+                "userLimit": 50,
+                "fileImportTimeout": 90000
```

Comment:
Let's update the .j2 file and default.json in framework as well and link the PRs here.

Reply:
Updated with [cfb98d1](https://github.com/wso2/identity-apps/pull/5939/commits/cfb98d12e9c1a1016a01ebb9159e28612a53880a)